app/_locales/en/messages.json

```diff
@@ -348,6 +348,18 @@
     "message": "Punycode version: $1",
```

Comment:
This will be merged in https://github.com/MetaMask/metamask-extension/pull/35289